ocf/ocf.go

```diff
@@ -292,7 +292,7 @@ func (e *Encoder) Encode(v interface{}) error {
 // Flush flushes the underlying writer.
 func (e *Encoder) Flush() error {
 	if e.count == 0 {
-		return nil
+		return e.writer.Flush()
```

Comment:
This is not the correct fix for #196. Instead you should flush after writing the header.

Reply:
Fixed.